ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-main.yaml

```diff
@@ -77,7 +79,7 @@ tests:
     owner: openstack-k8s-operators
     product: ocp
     timeout: 1h0m0s
-    version: "4.10"
+    version: "4.12"
   steps:
     workflow: openstack-k8s-operators-test-build-deploy-kuttl
```

Comment:
@viroel 

we missed to add `CEPH_HOSTNETWORK: "false"` here.

~~~
    env:
      CEPH_HOSTNETWORK: "false"
~~~

that's why we hit below issue in the first run

https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41611/rehearse-41611-pull-ci-openstack-k8s-operators-cinder-operator-main-cinder-operator-build-deploy-kuttl/1683637428847382528/artifacts/cinder-operator-build-deploy-kuttl/openstack-k8s-operators-gather/artifacts/api-resources/openstackcontrolplanes.yaml

~~~
    - lastTransitionTime: "2023-07-25T01:48:56Z"
      message: OpenStackControlPlane Glance in progress
      reason: Requested
      severity: Info
      status: "False"
      type: Ready
~~~

Reply:
fixed.